dialogflow-app.js

```diff
@@ -325,6 +325,9 @@ class DialogflowApp extends AssistantApp {
   getIncomingRichResponse () {
     debug('getIncomingRichResponse');
     const response = this.buildRichResponse();
+    if (response.event === undefined) {
+      delete response.event;
```

Comment:
Would this change anything? If it's `undefined` then why delete it?

Reply:
It was because of a falsy problem with the tests. So I'll fix in that side.